uaclient/status.py

```diff
@@ -392,8 +393,14 @@ def colorize_commands(commands: "List[List[str]]") -> str:
         if content:
             content += " && "
         content += " ".join(cmd)
+    # subtract 2 from print width to make room for space and backslash
+    wrapped_content = " \\\n".join(
```

Comment:
Sorry I misread that escape sequent, right we totally should have the trailing backslash to allow humans to cut-n-paste the commands. Along the same traing of thought, I think we need to make our output look more like this
```bash
{
   some long \
   command
}
# instead of 
{ some long \
  command }. 
```

Here's a patch of what I was thinking. See what you think or adapt as necessary. The reason I wanted to separate multi-line commands from the  leading/trailing brances is because we don't want cut-n-paste efforts with a mouse to have to worry about accidental inclusion of leading or trailing characters unrelated to the command.


```diff
diff --git a/uaclient/status.py b/uaclient/status.py
index 07eb68b..9824a25 100644
--- a/uaclient/status.py
+++ b/uaclient/status.py
@@ -1,7 +1,8 @@
 import enum
 import sys
+import textwrap
 
-from uaclient.defaults import BASE_UA_URL
+from uaclient.defaults import BASE_UA_URL, PRINT_WRAP_WIDTH
 
 try:
     from typing import Any, Dict, List, Optional, Tuple, Union  # noqa: F401
@@ -392,8 +393,21 @@ def colorize_commands(commands: "List[List[str]]") -> str:
         if content:
             content += " && "
         content += " ".join(cmd)
-    return "{color}{{ {content} }}{end}".format(
-        color=TxtColor.DISABLEGREY, content=content, end=TxtColor.ENDC
+    # subtract 4 from print width to account for leading and trailing braces
+    # and spaces.
+    wrapped_content = " \\\n".join(
+        textwrap.wrap(
+            content, width=(PRINT_WRAP_WIDTH - 4), subsequent_indent="  "
+        )
+    )
+    if "\n" in wrapped_content:
+        prefix = "{\n  "
+        suffix = "\n}"
+    else:
+        prefix = "{ "
+        suffix = " }"
+    return "{color}{prefix}{content}{suffix}{end}".format(
+        color=TxtColor.DISABLEGREY, content=wrapped_content, end=TxtColor.ENDC, prefix=prefix, suffix=suffix
     )
 

```

Reply:
Very good point! I'll work this in

Reply:
This is in the latest commit - ended up looking exactly like this, plus the associated test change.